packages/ui/src/components/DataMapper/debug/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>DataMapper Debugger</title>
+  </head>
+  <body>
+    <script type="module">
+      // Temporary workaround for @patternfly/react-topology using global
```

Comment:
is it still necessary? Which issue must be fixed to remove it? What about including the related patternfly issue in the comment?

Reply:
Probably not, if I'm not mistaken this should have been fixed in topology upstream. In any case, this is a debug component that it's not part of Kaoto, so we can skip it for now.